include/z64.h

```diff
@@ -207,7 +209,7 @@ typedef struct
     /* 0x1418 */ u8           fade_duration;
     /* 0x1419 */ u8           unk_1419; // transition related
     /* 0x141A */ u16          environment_time;
-    /* 0x141C */ u8           unk_141C;
+    /* 0x141C */ u8           richardIsLost;
```

Comment:
For what it's worth this appears to have been called `dog_event_flag`, and we might want to use a similar name that mentions "dog" rather than richard which is too specific imo.

Reply:
its specifically used for Richard being lost though, its not like dog params which is used for any dog that is following you. Ill try to think of something that follows this more, but I personally dont see an issue with making it about richard

Reply:
I don't mean that it's not specific for richard, but the fact that the dog is called richard in the game's text is a detail that doesn't really belong in the code itself.